pkg/node/node.go

```diff
@@ -61,6 +69,16 @@ type Node struct {
 	cluster *clusterConfiguation
 }
 
+// Fullname returns the node's full name including the cluster name if the
+// cluster name has been specified
+func (n *Node) Fullname() string {
```

Comment:
I don't understand this. How can you know if the cluster name has been specified or not if you are assigning a default value ("default") to the CLI flag?

Reply:
We can't. I can clarify the comment to say: specified a value other than the default value.

Reply:
fixed

Reply:
There was nothing wrong with the documentation, what I was trying to say was if the user uses the cluster name "default" then he will not see in the controllers outputs for example in `cilium status`.

Reply:
Yes, that's intentional. Do you see a problem with that? You would not expect to run multiple clusters with the name default.